app/[slug]/page.tsx

```diff
@@ -74,78 +70,11 @@ export default function Group() {
                 <p>{e.name}</p>
                 <div className="flex items-center self-center gap-2">
                   {e.links?.map((e: urlDTO, i: number) => {
```

Comment:
not breaking, but difficult readability: iterating over "e" when items inside are also "e"?